.github/styles/kong/OxfordComma.yml

```diff
@@ -0,0 +1,12 @@
+# Warning: OxfordComma
```

Comment:
Do filenames need to be `TitleCase`? Case sensitivity can cause issues on some filesystems so we tend towards all lowercase with `-` separators